packages/contracts/deploy-config/devnetL1-template.json

```diff
@@ -44,6 +44,7 @@
   "validatorManagerTrustedValidator": "0x70997970C51812dc3A010C7d01b50e0d17dc79C8",
   "validatorManagerMinRegisterAmount": "0x32",
   "validatorManagerMinActivateAmount": "0x64",
+  "validatorManagerMptFirstOutputIndex": "0xffff",
```

Comment:
We already set l2GenesisKromaMPTTimeOffset to 4 in devnet (see above), so you should set it accordingly.

Reply:
I'm not sure what to do.
The validatorPoolTerminateOutputIndex is 32 and the validatorManagerMptFirstOutputIndex needs to be larger than that for v2 to make sense. Rather than correcting all of this, I chose to leave the actual validatorPoolTerminateOutputIndex value large enough to be meaningful. Why not inject validatorPoolTerminateOutputIndex only in the tests that need it?